projects/ngx-signature-pad/src/lib/ngx-signature-pad.component.ts

```diff
@@ -62,26 +66,30 @@ export class NgxSignaturePadComponent implements OnInit, OnChanges {
       width: '100%'
     });
     this.overlayRef.attach(this.portal);
-    this.initBigPad();
-    // #region Copy miniScreen's content to fullScreen
-    const { width: miniScreenWidth, height: miniScreenHeight } = this.options;
-    const ctx = this.bigCanvas.getContext('2d');
-    ctx.save();
-    ctx.translate(this.fullScreenWidth, 0);
-    ctx.rotate((90 * Math.PI) / 180);
-    ctx.drawImage(
-      this.smallCanvas,
-      0,
-      0,
-      miniScreenWidth,
-      miniScreenHeight,
-      0,
-      0,
-      this.fullScreenHeight,
-      this.fullScreenWidth
-    );
-    ctx.restore();
-    // #endregion
+
+    setTimeout(() => {
```

Comment:
Why you need `setTimeout`?

Reply:
in this case it is because `this.nspBig.nativeElement` that is inside `this.initBigPad();` is read before the element is created on the screen. The `setTimeOut({}, 0)` makes this process asynchronous and makes angular detect the change in this browser MacroTask. same like in this case https://angular.io/errors/NG0100